cmd/podman/inspect.go

```diff
@@ -157,7 +163,12 @@ func iterateInput(ctx context.Context, c *cli.Context, args []string, runtime *l
 					inspectError = errors.Wrapf(err, "error getting libpod container inspect data %s", ctr.ID())
 					break
 				}
-				data, err = shared.GetCtrInspectInfo(ctr, libpodInspectData)
+				artifact, err := getArtifact(ctr)
+				if inspectError != nil {
```

Comment:
Did you mean to test "err" here in place of "inspectError"?

Reply:
good catch